src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -1113,67 +1101,50 @@ public static function updateCategory(array $item, array $meta = null): int
         return $updated;
     }
 
-    /**
-     * Update an existing comment
-     *
-     * @param array $item The new data.
-     *
-     * @return int
-     */
-    public static function updateComment(array $item): int
+    public static function updateComment(array $item)
     {
-        // update category
-        return BackendModel::getContainer()->get('database')->update(
-            'blog_comments',
-            $item,
-            'id = ?',
-            [(int) $item['id']]
+        $entityManager = BackendModel::get('doctrine.orm.default_entity_manager');
+        $repository = $entityManager->getRepository(Comment::class);
+        $comment = $repository->find($item['id']);
+
+        if (!$comment instanceof Comment) {
+            return;
+        }
+
+        $comment->update(
+            $item['author'],
+            $item['email'],
+            $item['website'],
+            $item['text'],
+            $comment->getType(),
+            $item['status'],
+            $comment->getData()
         );
+
+        $entityManager->flush($comment);
     }
 
-    /**
-     * Updates one or more comments' status
-     *
-     * @param array  $ids    The id(s) of the comment(s) to change the status for.
-     * @param string $status The new status.
-     */
-    public static function updateCommentStatuses(array $ids, string $status): void
-    {
-        // make sure $ids is an array
-        $ids = (array) $ids;
+    public static function updateCommentStatuses(array $ids, string $status): void{
```

Comment:
enter before the {

Reply:
Fixed in 18adc30c0965878417a31b49881be34846e4911a